src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java

```diff
@@ -147,7 +147,9 @@ public void updateIssuesTable() {
                 .map(scanManager -> scanManager.getFilteredScanIssues(FilterManagerService.getInstance(mainProject), selectedNodes))
                 .flatMap(Collection::stream)
                 .collect(Collectors.toSet());
-        issuesTable.updateIssuesTable(issueSet);
+
+        Set<String> selectedNodeNames = selectedNodes.stream().map(node -> node.toString()).collect(Collectors.toSet());
```

Comment:
Just to get rid of the IDE warning:
```suggestion
        Set<String> selectedNodeNames = selectedNodes.stream().map(DefaultMutableTreeNode::toString).collect(Collectors.toSet());
```